integration-cli/docker_cli_events_test.go

```diff
@@ -230,6 +230,30 @@ func (s *DockerSuite) TestEventsImageUntagDelete(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestEventsImageTag(c *check.C) {
+	time.Sleep(time.Second * 2) // because API has seconds granularity
```

Comment:
ok, I see now, IMO this test shouldn't really test how many events are out there but ONLY that a tag event exists here (making the sleep above unnecessary). I don't really see the point in making sure only this event is triggered but can be only me :)